.gitignore

```
@@ -1,3 +1,4 @@
 .idea/*
 *.iml
 target/*
+*.log
```

add blank line at the end of the file, so github wont mark it with ⛔ symbol
```suggestion
*.log

```